### PR TITLE
[ALLUXIO-2798] Use precondition check chaining in JobConfInfo

### PR DESCRIPTION
**Pull request by lay139217**
https://alluxio.atlassian.net/browse/ALLUXIO-2798

The preconditions check in setOutputFile(String outputFile) can be chained to save a statement:
mOutputFile = Preconditions.checkNotNull(outputFile);

### CODE REVIEW

**Comment by aaudiber** [comment]
alluxio-bot, check this please

**Comment by aaudiber** [comment]
alluxio-bot, check this please

**Comment by apc999** [comment]
@lay139217 could you sign [the contribution license agreement](http://cla.alluxio.org/)?

File: core/common/src/main/java/alluxio/job/JobConf.java [hunk 1]
```diff
@@ -32,7 +33,8 @@
    * @param outputFilePath the path of the output file
    */
   public JobConf(String outputFilePath) {
-    mOutputFilePath = outputFilePath;
+    mOutputFilePath = Preconditions.checkNotNull(outputFilePath);
+    //mOutputFilePath = outputFilePath;
```
Comment (aaudiber) [inline core/common/src/main/java/alluxio/job/JobConf.java]: Can you remove the commented-out line?

**Comment by yupeng9** [comment]
jenkins, test this please

**Comment by yupeng9** [comment]
@lay139217 Thanks for the contribution. There's a check style issue on the import order. Can you address that?

**Comment by aaudiber** [comment]
@lay139217 Are you still planning on working on this?

**Comment by yupeng9** [comment]
@lay139217  Feel free to reopen this PR when you have the style fixed.

**Pull request closed without merging**